.docker/source/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN \
     rm -rf /var/lib/apt/lists/*
 
 # Build the workspace
-RUN . /opt/ros/${ROS_DISTRO}/setup.sh &&\
+RUN cd .. && . /opt/ros/${ROS_DISTRO}/setup.sh &&\
```

Comment:
To make it clearer where you are changing directory to, could you use the variable ROS_DISTRO?

Reply:
There's no need to use `ROS_DISTRO`, the previous current directory is `/root/ws_moveit/src`

Reply:
What I meant was to use `ROS_UNDERLAY`:

`cd $ROS_UNDERLAY/..`

I'm sorry, not ROS_DISTRO.